compose.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  app:
+    image: node:lts-alpine
+    working_dir: /app
+    depends_on:
+      db:
+        condition: service_healthy
+        restart: true
+    environment:
+      DATABASE_URL: postgres://postgres:mysecretpassword@db:5432/mydatabase?sslmode=disable
+      GOOGLE_CLIENT_ID: google-client-id
+      JWT_SECRET: jwt-secret
+      SERVER_RUL: server-url
```

Comment:
These three environment variables are mock values. They are here to bypass the checks and allow the app to spin up. If you intend to work on the authentication route, you'll need to override them when spinning up the Docker containers.

Reply:
this should be a comment in the yml file itself :spiral_notepad:

Reply:
Got you.